### PR TITLE
Homework29: Aksana Batyrova

### PR DESCRIPTION
**Pull request by AksanaBatyrova**
image

### CODE REVIEW

File: homework29/flask-app/readme.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+### Run the Docker image
+
+```
+docker run -p 8866:8866 --name my-python-app my-python-app
+```
+
+### Viewing the Application
+
+```
+http://localhost:5001
```
Comment (Stml89) [inline homework29/flask-app/readme.md:28]: 5001 port?

**Review by Stml89 — CHANGES_REQUESTED** [review]
Seems like you forgot to commit `index.html` file

File: homework29/flask-app/readme.md (hunk 1 above)
Comment (AksanaBatyrova) [reply homework29/flask-app/readme.md]: fixed

**Comment by AksanaBatyrova** [comment]
> Seems like you forgot to commit `index.html` file

removed .html from .gitignore, should be available now

**Review by Stml89 — APPROVED** [review]

**Pull request merged**